.github/workflows/ci-build-from-stable.yml

```
@@ -0,0 +1,40 @@
+name: Build from Stable
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - master
+
+env:
+  SCHEME: scheme
+  IDRIS2_TESTS_CG: chez
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install build dependencies
+        run: |
+          sudo apt-get install -y chezscheme
+          echo "$HOME/.idris2/bin" >> $GITHUB_PATH
+      - name: Install stable version
+        run: |
+          wget https://www.idris-lang.org/idris2-src/idris2-latest.tgz
+          tar -xf idris2-latest.tgz
+          cd Idris2*
+          make bootstrap SCHEME=scheme
+          make install
+          cd ..
+          rm -rf ./Idris2*
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Build current from stable
+        run: make all && make install
```

Can we try the following? Basically: how much of the problem can be solved by "use the updated `base`"?

```suggestion
        run: make libs && make install-libs && make all && make install
```

---

At the moment `make libs` sets `IDRIS2` variable to `../../${TARGET}` (relatively to each lib's dir) where `TARGET` is `build/exec/idris2`.

So to do what you suggest, we need change it, i.e. to make `IDRIS2` variable to be tunable for lib targets so that we can set it to point to idris2 which is installed at previous steps.

---

@gallais I think your suggestion changes the purpose of the CI step (which I am not saying is necessarily a bad thing).

What I have PRed here is a check that the current install instructions (https://github.com/idris-lang/Idris2/blob/master/INSTALL.md#3-installing-with-an-existing-idris-2) are not broken by the incoming PR.

Your suggested change is more inline with a check that there is some possible path forward, but potentially only given a deviation from the current build instructions.